### ISSUE TITLE
How to join a private group/channel

### ISSUE THREAD

**Issue by stoehrm5** [EionRobb/purple-rocketchat#13]
I was able to setup pidgin + rocket chat successfully. 

What works:

- get room list
- direct message to other users
- join public channels

What does not work:

- join an existing private room/channel.

Is this possible at all or did I just not find the documentation?

**Comment by lproven** [comment]
I have the same issue.

**Comment by bbrauns** [comment]
It seems that you have to specify the rocket.chat `roomid` like `Llm5D2wEtbcxBLZmp` on the join chat dialog. Unfortunately the `roomid` is only visible via mongodb queries

**Comment by Mi-La** [comment]
I just tried to create private channel just for myself via webapp and then I was able to join just by writing the channel name to field in "add chat". Also make sure that you choose RC account in case you have multiple accounts.

**Comment by lazyest** [comment]
need some solution here too :(

**Comment by lazyest** [comment]
I found the possible way to get room_id but its not helping me
you can download user data in profile and in zip-file there will be a channels.json
Please confirm if someone get a result with.

**Comment by arabusov** [comment]
@lazyest Hello, for me it works fine. I add a chat in the main menu, and specify only room ID without room name --- it is obtained automatically

**Comment by lazyest** [comment]
Yeah, me too ;) So I think this can be described as an official way of doing this. Or maybe its possible to do some more easy way in future.
